### ISSUE TITLE
Homebrew version

### ISSUE THREAD

**Issue by vielmetti** [errordeveloper/kxd#2]
Add a homebrew version so this is as simple as "brew install kxd".

**Comment by errordeveloper** [comment]
Yes, that'd be nice to have, but it's a little to early,  the project is not mature enough.
